### ISSUE TITLE
add default license text

### ISSUE THREAD

**Issue by graybeal** [ncbo/bioportal-project#286]
If there is no License Information for an ontology, BioPortal doesn't display that parameter on the summary page. This is sometimes confusing for users who do not realize the general terms apply, which say under "Use of Ontologies":

> In general, any ontology submitted to Our Sites will be freely available for public use, unless the content is marked as private (in which case it is not shared via Our Sites) or the Licensing Attribute is defined in the metadata for that ontology.

So! If there is no License Information, create a License Information entry with the following information: 

> This ontology does not specify LIcense Information. Therefore (per the BioPortal Terms of Use), the ontology is freely available for public use, so long as its Visibility is "Public". (Note the Terms of Use specifies certain disclaimers on the use of all BioPortal ontologies that must still be observed.)

This will save people from wondering whether they can use it, and make the resource more FAIR by supporting FAIR Principle R1.1.

(If we want to be extra good we can cite a specific Creative Commons license, but that would take more time to figure out which one and decide if you wanted the Terms of Use to match the license, which might get tricky in the case of medical/clinical applications.)

(I guess I could do a pull request but I wouldn't be able to test it, so I'm thinking that's a bad idea.)

**Comment by jonquet** [comment]
Good point and idea. Indeed it would make sense to add a default sentence for any ontologies. In OntoPortal-Lirmm we have a cc:useGuidelines metadata property in addition of the dct:license property for license (which takes only URIs for better FAIR). We could use it.

**Comment by syphax-bouazzouni** [comment]
I would be more for the second proposition

> (If we want to be extra good we can cite a specific Creative Commons license, but that would take more time to figure out which one and decide if you wanted the Terms of Use to match the license, which might get tricky in the case of medical/clinical applications.) 

By default set the bellow license to all the ontologies, if no specific license is given 
<img width="627" alt="image" src="https://github.com/ncbo/bioportal-project/assets/29259906/0b869054-1d13-44ea-a753-b7898046c1dc">

**Comment by jonquet** [comment]
Unfortunatly, we can't decide the license for the ontology owner. This is something that has to be done by the depositor... assuming he his the owner and of not he/she must enquire to the owner. 
The only thing we can do is as @graybeal suggest : Bio/agroPortal assumption is that its "free and publicly accessible".
